jekyll/_cci2/workflows.adoc

```diff
@@ -34,35 +34,45 @@ For example, if only one job in a workflow fails, you will know it is failing in
 Workflows may appear with one of the following states:
 
 [.table.table-striped]
-[cols=2*, options="header", stripes=even]
+[cols=3*, options="header", stripes=even]
 |===
-| State | Description
+| State | Description | Terminal State
 
 | RUNNING
 | Workflow is in progress
+| No
 
 | NOT RUN
 | Workflow was never started
+| No
```

Comment:
NOT RUN is a terminal state; there is no other state it can transition to from there.

```suggestion
| Yes
```